pkg/assembler/graphql/schema/certifyBad.graphql

```diff
@@ -127,7 +127,15 @@ extend type Query {
 
 extend type Mutation {
   "Adds a certification that a package, source or artifact is considered bad."
```

Comment:
minor: if i recall from conversations - the ID is not necessarily a required return value right? in that case, would we set it as ID or at least document that ID can be empty "" (note may need to check if it may trigger a same bug as #1050).

Reply:
I've implemented some of these changes on the new PRs for bulk ingestion and it does not trigger an error when its set to an empty "". Agree that it should be documented that it can be an empty ""

Reply:
Added comment.